src/icons/general/IconAnyTask.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+import { bdlGray62 } from '../../styles/variables';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it is not purely decorative for accessibility */
+    title?: string,
+    width?: number,
+};
+
+const IconAnyTask = ({ className = '', color = bdlGray62, height = 10, title, width = 11 }: Props) => (
+    <AccessibleSVG className={className} height={height} title={title} viewBox="0 0 10 11" width={width}>
```

Comment:
Hmm, not a good viewbox. We were working toward them being consistently exported, and they should be 16x16 / 32x32.

Reply:
Probably preferred 32x32 if shrinking it to desired size doesn't cause artifacts.